js/coinex.js

```diff
@@ -3879,6 +3880,95 @@ module.exports = class coinex extends Exchange {
         return rates;
     }
 
+    async fetchBorrowInterest (code = undefined, symbol = undefined, since = undefined, limit = undefined, params = {}) {
```

Comment:
if cross and isolated are both valid, you should probably have a `params.marginMode`